docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -0,0 +1,210 @@
+
+`<snapshot>`::
+(Required, string)
+Comma-separated list of snapshot names to retrieve. Also accepts wildcards (`*`).
++
+* To get information about all snapshots in a registered repository, use a wildcard (`*`) or `_all`.
+* To get information about any snapshots that are currently running, use `_current`.
++
+NOTE: Using `_all` in a request fails if any snapshots are unavailable.
+Set <<get-snapshot-api-ignore-unavailable,`ignore_unavailable`>> to `true` to return only available snapshots.
+
+[role="child-attributes"]
+[[get-snapshot-api-request-body]]
+==== {api-request-body-title}
+
+[[get-snapshot-api-ignore-unavailable]]
+`ignore_unavailable`::
+(Optional, boolean)
+If `false`, the request returns an error for any snapshots that are unavailable. Defaults to `false`.
++
+If `true`, the request ignores snapshots that are unavailable, such as those that are corrupted or temporarily cannot be returned.
+
+`verbose`::
+(Optional, boolean)
+If `true`, returns all available information about a snapshot. Defaults to `true`.
++
+If `false`, omits additional information about the snapshot, such as version information, start and end times, and the number of snapshotted shards.
+
+[role="child-attributes"]
+[[get-snapshot-api-response-body]]
+==== {api-response-body-title}
+
+`snapshot`::
+Name of the snapshot.
+
+`uuid`::
+Universally unique identifier (UUID) of the snapshot.
+
+`version_id`::
+Build ID of the {es} version used to create the snapshot.
+
+`version`::
+{es} version used to create the snapshot.
+
+`indices`::
+Indices that are included in the snapshot.
+
+`start_time`::
+Date timestamp of when the snapshot creation process started.
+
+`start_time_in_millis`::
+The start time, in milliseconds, when the snapshot creation process started.
+
+`end_time`::
+Date timestamp of when the snapshot creation process ended.
+
+`end_time_in_millis`::
+The end time, in milliseconds, when the snapshot creation process ended.
+
+`duration_in_millis`::
+How long, in milliseconds, it took to create the snapshot.
+
+`failures`::
+Lists any failures that occurred when creating the snapshot.
+
+`shards`::
+Indicates the following shards information:
+* Total shards included in the snapshot
+* Failed shards that were not included in the snapshot
+* Successful shards that were included in the snapshot
+
+`state`::
++
+--
+(string)
+The snapshot `state` can be one of the following values:
+
+.Values for `state`
+[%collapsible%open]
+====
+`IN_PROGRESS`::
+  The snapshot is currently running.
+
+`SUCCESS`::
+  The snapshot finished and all shards were stored successfully.
+
+`FAILED`::
+  The snapshot finished with an error and failed to store any data.
+
+`PARTIAL`::
+  The global cluster state was stored, but data of at least one shard was not stored successfully.
+  The `failures` section of the response contains more detailed information about shards
+  that were not processed correctly.
+
+`INCOMPATIBLE`::
```

Comment:
I think we can leave this one out. I don't think this is ever returned in master or `7.x` at this point. Also, it's a little confusing to state:

>  The snapshot was created with an old version of {es} and is incompatible with the current version of the cluster.

This is true yes. But historically we would only show this for ES `1.x` snapshots and not in general anyway => let's just drop the documentation for this and I'll look into removing the last bits of code referencing `INCOMPATIBLE` shortly.

Reply:
Sounds good @original-brownbear. I'll remove the `INCOMPATIBLE` value for `state`.